_posts/2018-10-08-Pushing-Images-to-AWS.md

```diff
@@ -0,0 +1,159 @@
+---
+
+layout: post
+title: "Pushing composed images to AWS"
+author: swalter
+tags: welder-web lorax-composer
+
+---
+
+Weldr aka. Composer can generate images suitable for uploading to Amazon
+Web Services, and starting an EC2 instance. The images have the right
+partition layout, and include cloud-init.
+
+## Prerequisites
+
+We'll use [Fedora 29](https://getfedora.org/) as our OS of choice for running this. Run
+this in its own VM with at least 8 gigabytes of memory and 40 gigabytes of disk space.
+[Lorax](http://weldr.io/lorax/) makes some changes to the operating system its running on.
+
+First install Composer:
+
+    $ sudo yum install lorax-composer cockpit-composer cockpit composer-cli
+
+Next make sure to turn off [SELinux](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/5/html/deployment_guide/ch-selinux) on the system. Lorax doesn't yet work properly with
+SELinux running, as it installs an entire OS image in an alternate directory:
+
+    $ sudo setenforce 0
+    $ sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/' /etc/selinux/config
```

Comment:
This should be Permissive, not disabled. With it in Permissive mode it can still apply selinux labels when building, with it disabled it cannot.

Reply:
Good catch. Fixed.